docs/reference/modules/asset.md

```diff
@@ -0,0 +1,205 @@
+
+#### Example
+
+<AposCodeBlock>
+
+```sh
+APOS_RELEASE_ID='2022-01-01' node app @apostrophecms/asset:build
+```
+</AposCodeBlock>
+
+### `clear-cache`
+Some changes to the webpack configuration will not be detected automatically and will cause incorrect output. While this is rare, the caches for webpack can be emptied from the command line using the `clear-cache` task.
+
+#### Example
+
+<AposCodeBlock>
+
+```sh
+node app @apostrophecms/asset:clear-cache
+```
+</AposCodeBlock>
+
+## Webpack configuration
+
+The webpack configuration can be extended from any module. The `asset` module exposes three different properties for accomplishing this.
+
+## Webpack options
+|  Property | Type | Description |
+|---|---|---|
+| [`extensions`](#extensions) | Object | Adds extra processing functionality to webpack. |
+| [`extensionOptions`](#extensionoptions) | Object | Adds functionality to any extensions added through the `extensions` option. |
+| [`bundles`](#bundles) | Object | |
+
+### `extensions`
+The `extensions` option allows the addition of custom methods for processing the Javascript and SCSS in the `ui/src` folder. The schema used within the option conforms to that used for configuring webpack, which you can learn more about [here](https://webpack.js.org/configuration/). This property can take either a simple object with a name property or a function that returns an object. 
+
+#### `extensions` simple object
+
+#### Example
+
+Modification of the `resolve` property of the webpack configuration using a simple object. New values will be merged with existing values:
+
+<AposCodeBlock>
+
+```javascript
+const path = require("path");
+
+module.exports = {
+ // ...
+ webpack: {
+   extensions: {
+     modifyResolve: {
+       resolve: {
+         extensions: [ '.jsx' ],
+         alias: {
+           'Utilities': path.join(process.cwd(), 'lib/utils')
+         }
+       }
+     }
+   }
+ },
+ // ...
+};
+
+```
+<template v-slot:caption>
+modules/my-module/index.js
+</template>
+</AposCodeBlock>
+
+::: note
+Make sure the new values are of the same type as the existing values, or they will replace the existing values, e.g., the webpack configuration `extensions` property takes an array, whereas `alias` takes an object.
```

Comment:
I'm not sure I understand the significance of "or they will replace the existing values." I think your point here is more that the new values must be of the right type or they won't make sense to webpack.

Reply:
I don't know webpack well enough to know if there is a case where a property could take, for example, a string or an array of strings. They will merge if the existing value is an array and you supply an array. If you supply a string, the existing array will be deleted. Should we just delete the note?